frontend/src/components/SeedlotRegistrationSteps/CollectionStep/index.tsx

```diff
@@ -136,29 +180,36 @@ const CollectionStep = (
         <Row className="collector-agency-checkbox">
           <Column lg={12}>
             <Checkbox
-              id="collection-form-applicant"
+              id={fieldsConfig.checkbox.name}
               name={fieldsConfig.checkbox.name}
+              ref={(el: HTMLInputElement) => addRefs(el, fieldsConfig.checkbox.name)}
               labelText={fieldsConfig.checkbox.labelText}
-              defaultChecked
-              onChange={
-                (event: React.ChangeEvent<HTMLInputElement>) => collectorAgencyisChecked(event)
-              }
+              checked={isChecked}
+              onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                collectorAgencyIsChecked(e);
+              }}
             />
           </Column>
         </Row>
         <Row>
           <Column sm={12} md={4} lg={6}>
```

Comment:
Didn't noticed this before, sorry... But the column's sizes are a bit different with carbon, please follow this table: 
![image](https://user-images.githubusercontent.com/105936322/231773670-940826e1-0a2f-4fc5-a6d9-28f2178eb9e2.png)
Leaving only this markup, but please fix it on the other columns as well.